parachain/src/chain_spec/rococo.rs

```diff
@@ -1,23 +1,20 @@
 use cumulus_primitives_core::ParaId;
 use hex_literal::hex;
 use rococo_runtime::{
-	AccountId, EthereumHeader,
```

Comment:
ditto. Seem comment above about formatting.

Reply:
https://github.com/Snowfork/polkadot-ethereum/pull/442/commits/a31d23b3ae9b5d3f5a697ddb62b138c0c3225ad5 and https://github.com/Snowfork/polkadot-ethereum/pull/442/commits/8f9accfdf79d7b5bf931ae623a8eb55fbfc8af3c